### PR TITLE
Added new doc - specifications to set up subnet

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
- Added new document - Specifications to set up Subnet on mainnet

### CODE REVIEW

**Review by obycode — COMMENTED** [review]
It looks like the VM specifications were updated to lower requirements, but the hardware specs were not lowered accordingly.

File: docs/miner-resource-recommendation.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- Annual cost is approximately $10890 per year
+
+## Hardware Specifications
+
+The following are the hardware specifications for running a subnet on the mainnet.
+
+### CPU
+
+- Processing speed greater than or equal to 2.8 GHz
+- CPU should be greater than or equal to 12 Core, 24 Threads
```
Comment (saralab) [inline docs/miner-resource-recommendation.md:46]: CPU greater than or equal to 4 vCPUs

File: docs/miner-resource-recommendation.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### RAM
+
+- Preferred memory greater than or equal to 128GB
+- Minimum: 48GB
```
Comment (saralab) [inline docs/miner-resource-recommendation.md:51]: Minimum: 16 GB

File: docs/miner-resource-recommendation.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+The following are the hardware specifications for running a subnet on the mainnet.
+
+### CPU
+
+- Processing speed greater than or equal to 2.8 GHz
+- CPU should be greater than or equal to 12 Core, 24 Threads
+
+### RAM
+
+- Preferred memory greater than or equal to 128GB
```
Comment (saralab) [inline docs/miner-resource-recommendation.md:50]: Delete this

File: docs/miner-resource-recommendation.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- Minimum: 48GB
+- Motherboard capacity greater than or equal to 256GB
```
Comment (saralab) [inline docs/miner-resource-recommendation.md:52]: Delete

**Review by saralab — APPROVED** [review]

**Pull request merged**